enigma-core/app/src/wasm_u/wasm.rs

```diff
@@ -374,6 +374,22 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_sc_encryption() {
+        let (mut db, _dir) = create_test_db();
+        let message = b"Enigma";
+
+        compile_deploy_execute(
+            &mut db,
+            "../../examples/eng_wasm_contracts/encryption",
+            generate_contract_address(),
+            "construct()",
+            &[],
+            "encrypt_decrypt(bytes)",
+            &[Token::Bytes(message.to_vec())]
+        );
+    }
+
```

Comment:
The meaning of test is not just invoke the functionality, but to compare the results with the expected ones.

Reply:
The purpose of this test is to verify encryption/decryption, not to transform a value and verify the output. The contract will panic if the value isn't decrypted correctly and the test will fail. 

I'm more than happy to capture the decrypted output and assert again in the test. However, please understand that the reason why I didn't do it is because it is redundant. You should not assume that I don't know what a test is or that I'm being sloppy. It's hard to get all the subtleties correct when adding to an existing codebase.

Reply:
Oh, you are right, sorry, my comment was wrong.